.github/command-docs.md

```diff
@@ -77,23 +77,24 @@
-| .when        | Explain the absence of release dates or ETAs                                                                   | ---                                   |
-| .thumb       | Answers the big question, will it have FEATURE?                                                                | .willithave                           |
-| .xbox        | Short response + link to NOTAM for xbox marketplace                                                            | .xboxmarketplace                      |
+| Command           | Description                                                                                                    | Alias                                 |
+|:------------------|:---------------------------------------------------------------------------------------------------------------|:--------------------------------------|
+| .docsearch        | Provides a link to the FlyByWire documentation, either a general link, or a link for a specific search         | .documentation <br/> .docs <br/> .doc |
+| .donate           | Provides a link to the open collective                                                                         | ---                                   |
+| .goldenrules      | Provides an image describing the golden rules an Airbus pilot should follow                                    | .golden <br/> .gr                     |
+| .headwind         | Provides link to the Headwind Discord server                                                                   | .hw                                   |
+| .installer        | Provides link to the new installer                                                                             | ---                                   |
+| .latlong          | Provides a cheat sheet for conversion between Latitude and longitude coordinates between short and long format | .llfix                                |
+| .msfsdisc         | Provides link to Microsoft Flight Simulator discord server                                                     | .fsdisc <br/> .msfsdiscord            |
+| .qa               | Links to the Quality Assurance docs page                                                                       | ---                                   |
+| .roadmap          | FBW Roadmap                                                                                                    | .goals                                |
+| .salty            | Provides link to salty discord server                                                                          | .sal <br/> .ninjo                     |
+| .synaptic         | Provides link to synaptic discord server                                                                       | .syn                                  |
+| .runsimplecommand | Runs a simple command created by the Moderators. These are temporary commands for simple messages.             | .sc <br/> .rsc                        |
```

Comment:
If this is to go in the general category, the `category:` within the command needs to be set to general and the file should be placed under `src/commands/general`

If the category in the command is `moderation` it will appear under the moderation tab in the help DM embeds